jenkins/greenlight/Jenkinsfile

```diff
@@ -29,8 +32,22 @@ node {
         openshift.withProject() {
 
           echo "Building the application artifacts ..."
-          def build = openshift.selector("bc", "${ANGULAR_BUILD_CONFIG}")
-          build.startBuild().logs("-f")
+          def buildConfig = openshift.selector("bc", "${ANGULAR_BUILD_CONFIG}")
```

Comment:
Have a look at the changes here; https://github.com/bcgov/TheOrgBook/pull/796/files#diff-943414d5255d30e08b192bc16c461526

They should help clean up this pipeline a bit more.

Reply:
I applied the changes you suggested, thanks!
Pipeline tested and completed successfully.